bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/BaseBridgeHandler.java

```diff
@@ -53,4 +55,18 @@ public Thing getThingByUID(ThingUID uid) {
     public Bridge getThing() {
         return (Bridge) super.getThing();
     }
+
+    /**
+     * Creates a bridge builder, which allows to modify the bridge. The method
+     * {@link BaseThingHandler#updateThing(Thing)} must be called to persist the changes.
+     *
+     * @return {@link BridgeBuilder} which builds an exact copy of the bridge (not null)
+     */
+    @Override
+    protected ThingBuilder editThing() {
```

Comment:
One last comment: Should we return BridgeBuilder, here?
As BridgeBuilder is a child of ThingBuilder, I assume it works.